enums/src/macros.rs

```diff
@@ -15,24 +15,17 @@ macro_rules! mk_enum {
 macro_rules! mk_get_language {
     ( $( ($camel:ident, $name:ident) ),* ) => {
         pub fn get_language(lang: &LANG) -> Language {
-              match lang {
-                  LANG::Java => tree_sitter_java::language(),
-                  LANG::Typescript => tree_sitter_typescript::language_typescript(),
-                  LANG::Tsx => tree_sitter_typescript::language_tsx(),
-                  LANG::Javascript => tree_sitter_javascript::language(),
-                  LANG::Python => tree_sitter_python::language(),
-                  LANG::Preproc => tree_sitter_preproc::language(),
-                  LANG::Ccomment => tree_sitter_ccomment::language(),
-                  LANG::Cpp => tree_sitter_mozcpp::language(),
-                  LANG::Mozjs => tree_sitter_mozjs::language(),
-                  _ => match lang {
-                    $(
-                        LANG::$camel => {
-                            extern "C" { fn $name() -> Language; }
-                            unsafe { $name() }
-                        },
-                    )*
-                }
+            match lang {
```

Comment:
Could you keep the old indentation? Otherwise it looks like you're changing more than you actually are